sdk/dotnet/Pulumi.Analyzers/Pulumi.Analyzers/tools/install.ps1

```diff
@@ -0,0 +1,58 @@
+param($installPath, $toolsPath, $package, $project)

```

Comment:
these are the provided pieces to basically bypass .net core and get VS to load up these analyzers in standard